sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -2019,6 +2019,75 @@ def get_page_ranges( # type: ignore
             process_storage_error(error)
         return get_page_ranges_result(ranges)
 
+    @distributed_trace
+    def get_managed_disk_page_range_diff( # type: ignore
+            self, offset=None, # type: Optional[int]
+            length=None,  # type: Optional[int]
+            prev_snapshot_url=None,  # type: Optional[str]
+            **kwargs
+        ):
+        # type: (...) -> Tuple[List[Dict[str, int]], List[Dict[str, int]]]
+        """Returns the list of valid page ranges for a Page Blob or snapshot
+        of a page blob in managed disk account.
```

Comment:
managed disk account might be confusing, maybe consider:

Returns the list of valid page ranges for a managed disk or snapshot.